Examples/Tests/photon_photon_collisions/analysis_photon_photon_collisions.py

```diff
@@ -0,0 +1,200 @@
+    assert(np.all(is_close(total_energy, total_energy[0], rtol=5e-12)))
+
+def check_momentum_conservation():
+    total_momentum_x = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,2]
+    total_momentum_y = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,3]
+    total_momentum_z = np.loadtxt('diags/reducedfiles/ParticleMomentum.txt')[:,4]
+    assert(np.all(is_close(total_momentum_x, total_momentum_x[0])))
+    assert(np.all(is_close(total_momentum_y, total_momentum_y[0])))
+    assert(np.all(is_close(total_momentum_z, total_momentum_z[0])))
+
+def check_charge_conservation():
+    rho_max = np.loadtxt('diags/reducedfiles/RhoMaximum.txt')[:,2]
+    assert(is_close(rho_max, rho_max[0]).all())
+    #series = io.Series("diags/fields_particles/openpmd_%T.bp",io.Access.read_only)
+    #iterations = np.asarray(series.iterations)
+    #start = series.iterations[iterations[0]]
+    #rho_start = start.meshes["rho"][io.Mesh_Record_Component.SCALAR].load_chunk()
+    #end = series.iterations[iterations[-1]]
+    #rho_end = end.meshes["rho"][io.Mesh_Record_Component.SCALAR].load_chunk()
+    #series.flush()
+
+    filename_end = sys.argv[1]
+    filename_start = filename_end[:-4] + '0000'
+    ds_end = yt.load(filename_end)
+    ds_start = yt.load(filename_start)
+    field_data_end = ds_end.covering_grid(level=0, left_edge=ds_end.domain_left_edge,
+                                          dims=ds_end.domain_dimensions)
+    field_data_start = ds_start.covering_grid(level=0, left_edge=ds_start.domain_left_edge,
+                                              dims=ds_start.domain_dimensions)
+    rho_start = field_data_start["rho"].to_ndarray()
+    rho_end = field_data_end["rho"].to_ndarray()
+    assert(np.all(is_close(rho_start, rho_end)))
+
+def cross_section(E1_lab, E2_lab, theta):
+    s = E1_lab*E2_lab/(2.*m_e**2*c**4)*(1.-np.cos(theta))
+    beta = np.sqrt(1.-1./s)
+    factor1 = 0.5*pi*r_e**2*(1.-beta**2)
+    term1 = (3.-beta**4)*np.log((1.+beta)/(1.-beta))
+    term2 = 2*beta*(beta**2-2.)
+    factor2 = term1 + term2
+    sigma = factor1*factor2
+    return sigma
+
+def check_pair_rate(test):
+    if test == 'many_photons_test':
+        E1_lab, E2_lab, theta, dt, V  = get_input_parameters(test)
+        # number of real photons of species photonA
+        N1 = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[:,8]
+        # number of real photons of species photonB
+        N2 = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[:,9]
+        steps = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[:,0]
+        estimated_number_of_positrons = 0
+        for ts in steps.astype(int):
+            estimated_number_of_positrons += (1.-np.cos(theta))*c*N1[ts]*N2[ts]*cross_section(E1_lab, E2_lab, theta)/V*dt
+        obtained_number_of_positron = np.loadtxt('diags/reducedfiles/ParticleNumber.txt')[-1,11]
+        assert(is_close(obtained_number_of_positron, estimated_number_of_positrons, rtol=1e-1))
```

Comment:
Related to the comment above, could this be simply replaced by the following?
```suggestion
        assert(np.isclose(obtained_number_of_positron, estimated_number_of_positrons, rtol=1e-1, atol=0.))
```